quick-integration.hcl

```diff
@@ -0,0 +1,58 @@
+param "converge-bin-dir" {}
+
+param "image-name" {
+  default = "ubuntu"
+}
+
+param "image-tag" {
+  default = "latest"
```

Comment:
should use a specific version (`latest` is not a real version and while it is probably not likely with this particular workload, there is a risk that an updated ubuntu image could break the tests)